server/src/main/java/org/apache/iotdb/db/mpp/plan/expression/multi/builtin/helper/RoundHelper.java

```diff
@@ -0,0 +1,90 @@
+import org.apache.iotdb.db.mpp.plan.expression.multi.FunctionExpression;
+import org.apache.iotdb.db.mpp.plan.expression.multi.builtin.BuiltInScalarFunctionHelper;
+import org.apache.iotdb.db.mpp.transformation.api.LayerPointReader;
+import org.apache.iotdb.db.mpp.transformation.dag.column.ColumnTransformer;
+import org.apache.iotdb.db.mpp.transformation.dag.column.unary.scalar.RoundFunctionColumnTransformer;
+import org.apache.iotdb.db.mpp.transformation.dag.transformer.Transformer;
+import org.apache.iotdb.db.mpp.transformation.dag.transformer.unary.scalar.RoundFunctionTransformer;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
+import org.apache.iotdb.tsfile.read.common.type.TypeFactory;
+
+import java.util.List;
+
+import static org.apache.iotdb.db.mpp.plan.parser.ASTVisitor.checkFunctionExpressionInputSize;
+
+public class RoundHelper implements BuiltInScalarFunctionHelper {
+  @Override
+  public void checkBuiltInScalarFunctionInputSize(FunctionExpression functionExpression)
+      throws SemanticException {
+    checkFunctionExpressionInputSize(
+        functionExpression.getExpressionString(), functionExpression.getExpressions().size(), 1, 2);
+  }
+
+  @Override
+  public void checkBuiltInScalarFunctionInputDataType(TSDataType tsDataType)
+      throws SemanticException {
+    if (tsDataType.isNumeric()) {
+      return;
+    }
+    throw new SemanticException(
+        "Input series of Scalar function [ROUND] only supports numeric data types [INT32, INT64, FLOAT, DOUBLE]");
+  }
+
+  @Override
+  public TSDataType getBuiltInScalarFunctionReturnType(FunctionExpression functionExpression) {
+    return TSDataType.DOUBLE;
+  }
+
+  @Override
+  public ColumnTransformer getBuiltInScalarFunctionColumnTransformer(
+      FunctionExpression expression, ColumnTransformer columnTransformer) {
+    List<Expression> expressions = expression.getExpressions();
+    if (expressions.size() == 1) {
+      return new RoundFunctionColumnTransformer(
+          TypeFactory.getType(this.getBuiltInScalarFunctionReturnType(expression)),
+          columnTransformer,
+          0);
+    }
+    return new RoundFunctionColumnTransformer(
+        TypeFactory.getType(this.getBuiltInScalarFunctionReturnType(expression)),
+        columnTransformer,
+        Integer.parseInt(expressions.get(1).getExpressionString()));
+  }
+
+  @Override
+  public Transformer getBuiltInScalarFunctionTransformer(
+      FunctionExpression expression, LayerPointReader layerPointReader) {
+    List<Expression> expressions = expression.getExpressions();
+    if (expressions.size() == 1) {
+      return new RoundFunctionTransformer(
+          layerPointReader, this.getBuiltInScalarFunctionReturnType(expression), 0);
+    }
+    return new RoundFunctionTransformer(
+        layerPointReader,
+        this.getBuiltInScalarFunctionReturnType(expression),
+        Integer.parseInt(expressions.get(1).getExpressionString()));
```

Comment:
It is better we use attributeValue to represent places rather than using expression.